### DISCUSSION TITLE
could not find driver (Connection: sqlsrv, SQL: select * from [users]

### DISCUSSION THREAD

**Discussion by mawdho** [beyondcode/herd-community#536] [Q&A] [answered]
Hello there

I'm using Windows 11 and SQL Server

When I run in browser ```DB::table('users')->get(); ``` I got this ``` could not find driver (Connection: sqlsrv, SQL: select * from [users]```.

Could anybody help ?

**Comment by mawdho** [comment]
When I execute ```php artisan migrate``` or ```php artisan tinker``` it works but when I visit the route that contains a query I got the issue.

**Comment by rubendn** [comment, marked as answer]
Have you added the SQL Server extension DLL to the ext directory of the PHP version you are using and added it to php.ini?  The SQL Server extension is not included by default.

**Reply by mawdho** [reply]
Thank you
